### PR TITLE
Don't try to use test-css command in CI.

### PR DESCRIPTION
**Pull request by jdm**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22176)

### CODE REVIEW

File: etc/ci/check_intermittents.sh [hunk 1]
```diff
@@ -9,16 +9,14 @@ set -o nounset
...
-            --log-raw - \
-            --repeat "${REPEAT_COUNT}" \
-            "${test_name}" \
-            > intermittents.log \
-            < /dev/null
-    done < "etc/ci/former_intermittents_${test_type}.txt"
-done
+while read test_name; do
+    echo "  - Checking ${test_name}"
+    ./mach "test-${test_type}" \
```
Comment (KiChjang) [inline etc/ci/check_intermittents.sh]: Where is test_type defined after these changes?

**Comment by jdm** [comment]
Good catch!

**Pull request merged**